src/components/loading-button/loading-button.e2e.ts

```diff
@@ -0,0 +1,196 @@
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
+
+describe('limel-loading-button', () => {
+    let page: E2EPage;
+    let limelLoadingButton: E2EElement;
+    let innerButton: E2EElement;
+
+    describe('with a label', () => {
```

Comment:
Do we really want to duplicate the tests in detail for each prop? I think we can write the tests so that both `limel-button` and `limel-loading-button` are tested the same way?